ewutils.py

```diff
@@ -296,7 +299,11 @@ def databaseClose(conn_info):
 
 """ format responses with the username: """
 def formatMessage(user_target, message):
-	return "*{}*: {}".format(user_target.display_name, message)#.replace("@", "\{at\}")
+	# If the display name belongs to a raid boss, hide it while it's counting down.
+	if user_target.display_name in ewcfg.raid_boss_names and user_target.life_state == 2:
+		return "{}".format(message)
+	else:
+		return "*{}*: {}".format(user_target.display_name, message).replace("@", "\{at\}")
```

Comment:
seems like this would cause an error if a player changed their name to a raid boss', because member objects don't have a life_state attribute